docs/dev-setup.md

```diff
@@ -80,6 +80,10 @@ The following environment variables can be passed to configure the ironic:
 - IRONIC_FAST_TRACK - whether to enable fast_track provisioning or not
   (default true)
 - IRONIC_KERNEL_PARAMS - Kernel parameters to pass to IPA (default console=ttyS0)
+- IRONIC_INSPECTOR_VLAN_INTERFACES - VLAN interfaces to be included in introspection report
+       `all` - all VLANs on all interfaces, using LLDP information (default)
+       interface - all VLANs on an interface, using LLDP information
+       interface.vlan - a particular VLAN interface, not using LLDP
```

Comment:
Newlines are not rendered in markdown, you need at least commas between the items